src/main.ts

```diff
@@ -21,6 +21,13 @@ async function bootstrap() {
       'JWT-auth',
       )
   .build();
+  app.enableCors({
+    origin: [
+      "http://localhost:5173",
```

Comment:
Can we enable two other frontend ports? For more stable testing.
```suggestion
      "http://localhost:4173",
      "http://localhost:5173",
      "http://localhost:8080",
```